### ISSUE TITLE
The requirement to sell all bought gear during a round before it ends is rather awkward.

### ISSUE THREAD

**Issue by NancokPS2** [new-frontiers-14/frontier-station-14#1583]
### Money minimum for characters round start.
Join the game, spend money, end round without selling any bought gear/ships, you're now broke next round and can't even afford stuff to make more money.  
It also fucks over new players that don't know how to make a profit or don't know about the persistence of funds. Or simply someone that left mid-round for any reasons.  
  
It would be cool if there was at least a lower limit, so even if you go under, you don't have to make a new character next round to be able to leave the station without begging for a position on a ship.    

Simply if you end a round with less than 20K, you start with 20K next round.  
The next best would be to save the player's posessions and load them next time, but i know that's a WYCI and probably will have to wait until upstream's serialization stuff if anything.        
  
### Prevent loosing all investments if a player fails to sell their gear in time.
  
It is also sort of odd that players have to manually sell their ship at the end of the round, idk how easy it would be to make them auto-sell at the end (since there is no reason to not do so anyway). This is an issue because if ANYTHING happens to the player which makes them unable to sell their ship, any money "invested" that round is permanently lost. Including disconnecting before getting to a shipyard.

**Comment by whatston3** [comment]
Your description does not quite match your title.  Addressing the title: "New players are not communicated spesos persistence" - the guidebook's been updated in the pending upstream merge.  If you think what's there is insufficient or difficult to follow after the merge, happy to follow up on this.  The persistance of your bank account, depositing cash before shift end, and selling your ship before shift end are all repeatedly emphasized.  If you don't want to read the guidebook or ask for help in-game, then I don't know how to help you.

Regarding your other point - losing your money needs to be a possibility, or money carries no weight and can be spent frivolously.  If you cannot handle losing all of your funds as one character and either working station jobs for a fixed wage or crewing back up to your initial amount, you can create a new character - that _is_ your safety valve.  I do not think that will change in the meantime.

With regard to "saving the player's possessions", I don't know what you mean.  Loadouts give you a way to have a prepared inventory in a particular round at the cost of money.  If you mean saving player possessions so you start next round with whatever you had on-hand in the last round, this completely breaks any sort of in-game progression.

**Comment by ThatOneGoblin25** [comment]
You can export your character's look to a New Character and they start with 25k.

**Comment by NancokPS2** [comment]
> Your description does not quite match your title. Addressing the title: "New players are not communicated spesos persistence" - the guidebook's been updated in the pending upstream merge. If you think what's there is insufficient or difficult to follow after the merge, happy to follow up on this. The persistance of your bank account, depositing cash before shift end, and selling your ship before shift end are all repeatedly emphasized. If you don't want to read the guidebook or ask for help in-game, then I don't know how to help you.
> 
> Regarding your other point - losing your money needs to be a possibility, or money carries no weight and can be spent frivolously. If you cannot handle losing all of your funds as one character and either working station jobs for a fixed wage or crewing back up to your initial amount, you can create a new character - that _is_ your safety valve. I do not think that will change in the meantime.
> 
> With regard to "saving the player's possessions", I don't know what you mean. Loadouts give you a way to have a prepared inventory in a particular round at the cost of money. If you mean saving player possessions so you start next round with whatever you had on-hand in the last round, this completely breaks any sort of in-game progression.

That would indeed solve the first issue i had. However, if you can copy the character to get 25K again, there is 0 reason to make you be able to start with less than that, it would not, in any shape or form, make money not matter.  
  
Loosing money should indeed be a thing, doing so because you forgot to/couldn't (for some reason) sell your ship at the end of the shift, is not exactly intuitive nor fun.  
I am sure that this is just a side effect of being unable to do so automatically, which is why i brought it up.  
It is kinda odd that the rounds work like a gamble. If you perma die, you cannot reconnect or otherwise do not reach the place to sell your ship in time, you instantly loose all investment that round for no reason.

**Comment by whatston3** [comment]
> That would indeed solve the first issue i had. However, if you can copy the character to get 25K again, there is 0 reason to make you be able to start with less than that, it would not, in any shape or form, make money not matter.
> 
> Loosing money should indeed be a thing, doing so because you forgot to/couldn't (for some reason) sell your ship at the end of the shift, is not exactly intuitive nor fun. I am sure that this is just a side effect of being unable to do so automatically, which is why i brought it up. It is kinda odd that the rounds work like a gamble. If you perma die, you cannot reconnect or otherwise do not reach the place to sell your ship in time, you instantly loose all investment that round for no reason.
> 
> You can already loose money by simply failing to make enough profit, and it makes no sense that everyone is required to re-sell their gear at the end of their shift as an all or nothing situation, but again, this is probably a side effect of the lack of serialization/ability to auto-sell bought gear.

I've run a character with zero spesos to their name with some success.  I've played Pun Pun without a bank account and worked on a ship as a cook.  You can make it work, and playing a poor character can be a genuine challenge and lead to interesting roleplay.

The cash-out mechanic is entirely what makes it a gamble - this is not a bad thing.  Yes, you _can_ stay out and make just a little more money, but the longer you do that, the more you push your luck, the likelier it is that something bad happens - maybe you die, maybe you're robbed by pirates, maybe there's a power outage.  The point is that you choose how far to push that out, and (most times) you had the ability to sell off earlier.  The fact that you say it's an all-or-nothing implies that you wouldn't go back earlier, if only to break even.  Hell, if you crewed up with people you knew, you could get them to take your stuff and pay you back later.

In short:
* Problems with the guidebook?  Happy to talk about them.
* There are ways to mitigate risk, but risk is intrinsic to the game.
* Auto-selling ships/gear?  Unlikely to happen.

**Comment by NancokPS2** [comment]
So the gambling aspect is completely intended? Is this confirmed in any design doc or similar?  
I mean, what VALUE does it provide? How does it enhance the rest of the game?

**Comment by NancokPS2** [comment]
Seems like the game is simply not what i was looking for. I could leave this up for the QoL of making it impossible to start with less than 25K, but i imagine that if they wanted to implement that, they'd done so already.

**Comment by Cheackraze** [comment]
> Seems like the game is simply not what i was looking for, people simply seem to enjoy the risk reward more than i expected. I could leave this up for the QoL of making it impossible to start with less than 25K, but i imagine that if they wanted to implement that, they'd done so already.

You seem to be confused. Starting a new character will give that character 25k, so new players actually are not at all locked out. I coded the system with this explicitly in mind.

**Comment by NancokPS2** [comment]
> > Seems like the game is simply not what i was looking for, people simply seem to enjoy the risk reward more than i expected. I could leave this up for the QoL of making it impossible to start with less than 25K, but i imagine that if they wanted to implement that, they'd done so already.
> 
> You seem to be confused. Starting a new character will give that character 25k, so new players actually are not at all locked out. I coded the system with this explicitly in mind.

As i already said, due to that, there is no reason why a character would ever start with less than 25K, so it is either a WYCI or intentional for some reason i don't understand.
